<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>521</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Blockade Depth Histogram</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="MplWidget" name="mpl_hist" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="queryLabel">
       <property name="text">
        <string>Filter:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="autocompleteedit" name="sqlQueryLineEdit">
       <property name="minimumSize">
        <size>
         <width>420</width>
         <height>0</height>
        </size>
       </property>
       <property name="placeholderText">
        <string>ResTime &gt; 0.1 and BlockDepth &lt; 0.8</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="queryLabel_2">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Bins:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="binsSpinBox">
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="errorLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>autocompleteedit</class>
   <extends>QLineEdit</extends>
   <header>autocompleteedit</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>binsSpinBox</tabstop>
  <tabstop>updateButton</tabstop>
  <tabstop>sqlQueryLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>sqlQueryLineEdit</sender>
   <signal>returnPressed()</signal>
   <receiver>updateButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>408</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>491</x>
     <y>447</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
